### DISCUSSION TITLE
Background image rendered yesterday, not rendering today?

### DISCUSSION THREAD

**Discussion by MimsySith** [community/community#23561] [Other Feature Feedback, Questions, & Ideas]
<p>I’m having a problem in GitHub pages with the background image for my webpage not rendering. The funny thing is, it didn’t render day before yesterday, worked briefly yesterday, and is not rendering again today. I’m a bit stumped as to why it would work intermittently. It’s a .png file, all lowercase (I read the recent post from someone who had a similar issue). I tried making the file size very small just in case it was too big a file - nothing. It works fine from file on my computer, just not on GitHub pages.</p>
<p>This is the page:  <a href="https://mimsysith.github.io/Mimsy-portfolio/" class="inline-onebox" rel="noopener nofollow ugc">Home page - Miriam's Portfolio</a></p>
<p>And this is the repository:  <a href="https://github.com/MimsySith/Mimsy-portfolio" class="inline-onebox" rel="noopener nofollow ugc">GitHub - MimsySith/Mimsy-portfolio: Online CV/Portfolio for M Smith</a></p>
<p>(You can’t see any text in the page because the text is white. The background was dark).</p>
<p>Would really appreciate any input as to why this may be happening.</p>

**Comment by MimsySith** [comment]
<p>Note to admins - I have adjusted the “contact” page so that no links are active anymore. No private information.</p>

**Comment by jfullstackdev** [comment]
<p>yup, those external files failed to load as what you can see,</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/9/1/91acd5736e682ae390a5b6bf168a4557123d767c.jpeg" data-download-href="/uploads/short-url/kMHwJhfnFBQYdqRsYxRFffeAk8s.jpeg?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181077906-021daa06-da70-4536-9184-f918bdb920ab.jpeg" alt="image" data-base62-sha1="kMHwJhfnFBQYdqRsYxRFffeAk8s" width="690" height="373" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/9/1/91acd5736e682ae390a5b6bf168a4557123d767c_2_690x373.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/9/1/91acd5736e682ae390a5b6bf168a4557123d767c_2_1035x559.jpeg 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/9/1/91acd5736e682ae390a5b6bf168a4557123d767c_2_1380x746.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/9/1/91acd5736e682ae390a5b6bf168a4557123d767c_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1920×1040 227 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>so, the issue is on their end, you can specifically look at each error by going to Developer tools</p>

**Comment by MimsySith** [comment]
<p>No, this is very strange. According to the console, it is looking for an old filename that isn’t in my code anymore. It’s looking for background_image instead of background-image, even though I have changed the CSS stylesheet to the latter. There is nothing in my code that says background_image anymore. I recommitted the stylesheet just to make sure, but am getting the same error. Why would it be looking for an old filename? Does Github Pages not update the stylesheet with everything else?</p>

**Comment by jfullstackdev** [comment]
<p>I think, it’s now working</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/9/1/911c1d3a98ddd77dcaf94edbf23540849a9d657b.jpeg" data-download-href="/uploads/short-url/kHHsZiaOyQ7uhLCaJBUP1Fy93gn.jpeg?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181077908-cb5d39d5-2f42-4533-aaae-cd83b5096fb0.jpeg" alt="image" data-base62-sha1="kHHsZiaOyQ7uhLCaJBUP1Fy93gn" width="690" height="373" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/9/1/911c1d3a98ddd77dcaf94edbf23540849a9d657b_2_690x373.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/9/1/911c1d3a98ddd77dcaf94edbf23540849a9d657b_2_1035x559.jpeg 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/9/1/911c1d3a98ddd77dcaf94edbf23540849a9d657b_2_1380x746.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/9/1/911c1d3a98ddd77dcaf94edbf23540849a9d657b_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1920×1040 176 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>the errors are gone, except for one,</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/e/2/e210486e5d7fede45baa1e4fcb61c44b88eccbb2.jpeg" data-download-href="/uploads/short-url/wfQNft4Pjqq3E1HnvzJo8fVcuZA.jpeg?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181077909-eaf1f351-d40c-4c2e-bfe6-a44e591abd31.jpeg" alt="image" data-base62-sha1="wfQNft4Pjqq3E1HnvzJo8fVcuZA" width="690" height="373" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/e/2/e210486e5d7fede45baa1e4fcb61c44b88eccbb2_2_690x373.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/e/2/e210486e5d7fede45baa1e4fcb61c44b88eccbb2_2_1035x559.jpeg 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/e/2/e210486e5d7fede45baa1e4fcb61c44b88eccbb2_2_1380x746.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/e/2/e210486e5d7fede45baa1e4fcb61c44b88eccbb2_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1920×1040 185 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by jfullstackdev** [comment, marked as answer]
<aside class="quote no-group" data-username="MimsySith" data-post="4" data-topic="225435">
<div class="title">
<div class="quote-controls"></div>
 MimsySith:</div>
<blockquote>
<p>There is nothing in my code that says background_image anymore. I recommitted the stylesheet just to make sure, but am getting the same error. Why would it be looking for an old filename? Does Github Pages not update the stylesheet with everything else?</p>
</blockquote>
</aside>
<p>I checked it just now, you just edit that a few minutes ago, sometimes there is the delay on GitHub Pages, to update it online, so now it’s already working</p>
